tests/sentry/backup/test_imports.py

```diff
@@ -81,6 +85,14 @@ def export_to_tmp_file_and_clear_database(self, tmp_dir) -> Path:
         clear_database()
         return tmp_path
 
+    def import_empty_backup_file(self, import_fn):
```

Comment:
nit: let's make a new test class, `DatabaseResetTests`, that inherits from `ImportTestCase`, and move your `test_..._existing_models_in_..._scope` tests into that new class instead. Seems like the clearest way to group them, since the tests as they stand aren't really related to `Sanitization`.